src/controllers/posts/getPostsCohortController.ts

```diff
@@ -0,0 +1,43 @@
+import { Response, NextFunction } from 'express';
+import { CustomError } from '../../utils';
+import { getAllPostsCohortQuery } from '../../database';
+import { GetCohortPostsRequest } from '../../utils/types';
+
+const getPostsCohortController = async (
+  req: GetCohortPostsRequest,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    const { page } = req.query;
+    const offset = (Number(page || 1) - 1) * 10;
+    const { stringId } = req.params;
```

Comment:
Try to use meaningful variables.
I think `id` or `cohortId` is more meaningful than `stringId`